pyquil/api/_compiler.py

```diff
@@ -121,7 +121,7 @@ async def _translate(*args) -> qcs_sdk.TranslationResult:  # type: ignore
         return EncryptedProgram(
             program=translated_program["program"],
             memory_descriptors=_collect_memory_descriptors(nq_program),
-            ro_sources={parse_mref(mref): source for mref, source in translated_program["ro_sources"] or []},
```

Comment:
oof

Reply:
Considering we missed it, we must have only ever tested with programs without readout...? Very strange to only discover now.